docs/content/quatrix.md

```diff
@@ -0,0 +1,238 @@
+
+     rclone config
+
+This will guide you through an interactive setup process:
+
+```
+No remotes found, make a new one?
+n) New remote
+s) Set configuration password
+q) Quit config
+n/s/q> n
+name> remote
+Type of storage to configure.
+Choose a number from below, or type in your own value
+[snip]
+XX / Quatrix by Maytech
+   \ "quatrix"
+[snip]
+Storage> quatrix
+API key for accessing Quatrix account.
+api_key> jS2pbTDjp9VNlzq3PsSVgyxK2BgE423vcPV.N3CCmqyHiyCM
+Host name of Quatrix account.
+host> example.quatrix.it
+
+--------------------
+[remote]
+api_key = jS2pbTDjp9VNlzq3PsSVgyxK2BgE423vcPV.N3CCmqyHiyCM
+host = example.quatrix.it
+--------------------
+y) Yes this is OK
+e) Edit this remote
+d) Delete this remote
+y/e/d> y
+```
+
+Once configured you can then use `rclone` like this,
+
+List directories in top level of your Quatrix
+
+    rclone lsd remote:
+
+List all the files in your Quatrix
+
+    rclone ls remote:
+
+To copy a local directory to an Quatrix directory called backup
+
+    rclone copy /home/source remote:backup
+
+### API key validity
+
+API Key is created with no expiration date. It will be valid until you delete or deactivate it in your account.
+After disabling, the API Key can be enabled back. If the API Key was deleted and a new key was created, you can
+update it in rclone config. The same happens if the hostname was changed.
+
+```
+$ rclone config
+Current remotes:
+
+Name                 Type
+====                 ====
+remote               quatrix
+
+e) Edit existing remote
+n) New remote
+d) Delete remote
+r) Rename remote
+c) Copy remote
+s) Set configuration password
+q) Quit config
+e/n/d/r/c/s/q> e
+Choose a number from below, or type in an existing value
+ 1 > remote
+remote> remote
+--------------------
+[remote]
+type = quatrix
+host = some_host.quatrix.it
+api_key = jS2pbTDjp9VNlzq3PsSVgyxK2BgE423vcPV.N3CCmqyHiyCM
+--------------------
+Edit remote
+Option api_key.
+API key for accessing Quatrix account
+Enter a string value. Press Enter for the default (jS2pbTDjp9VNlzq3PsSVgyxK2BgE423vcPV.N3CCmqyHiyCM)
+api_key>
+Option host.
+Host name of Quatrix account
+Enter a string value. Press Enter for the default (some_host.quatrix.it).
+
+--------------------
+[remote]
+type = quatrix
+host = some_host.quatrix.it
+api_key = jS2pbTDjp9VNlzq3PsSVgyxK2BgE423vcPV.N3CCmqyHiyCM
+--------------------
+y) Yes this is OK
+e) Edit this remote
+d) Delete this remote
+y/e/d> y
+```
+
+### Modified time and hashes
+
+Quatrix allows modification times to be set on objects accurate to 1 microsecond.
+These will be used to detect whether objects need syncing or not.
+
+Quatrix does not support SHA1 type hashes, so you cannot use the `--checksum` flag.
+
+### Restricted filename characters
+
+File names in Quatrix are case sensitive and have limitations like the maximum length of a filename is 255, and the minimum length is 1. A file name cannot be equal to `.` or `..` nor contain `/` or `\`.
```

Comment:
You are using the standard encoding which encodes stuff like control characters. Is this intentional? Might be worth a mention here.